examples/post-sync/README

```diff
@@ -0,0 +1,135 @@
+# Sync Status Watch Controller
+
+A Kubernetes controller that monitors and reports sync error status for Config Sync's `RootSync` and `RepoSync` resources.
+
+## Prerequisites
+
+- Go 1.24+
+- Docker
+- kubectl
+- Access to a Kubernetes cluster with Config Sync installed
+- Kubernetes cluster (v1.20+ recommended)
+- Container registry access (public or private)
+
+### **Note**
+⚠️ **This component is not released with Config Sync**.  
+- It is **only compatible** with the **Config Sync API at `main` branch HEAD**.  
```

Comment:
This depends on from which the branch you're reading this file, right?

Reply:
Change to use `same branch` instead of `main branch`